test/format/failure_configs/CONFIG.WT-5637

```diff
@@ -0,0 +1,103 @@
+############################################
+#  RUN PARAMETERS: V2
```

Comment:
Converted the initial V1 CONFIG to V2, and disabled `backup.incremental`, `ops.rebalance`, and `ops.salvage` to pass the format run for this CONFIG.

Reply:
FYI, as long as `backup=0` is set, it doesn't matter what the incremental setting is.